protonvpn_cli/utils.py

```diff
@@ -29,21 +33,26 @@ def call_api(url, json_format=True):
 
     logger.debug("Initiating API Call: {0}".format(url))
 
+    # For manual error handling, such as in wait_for_network()
+    if not handle_errors:
+        response = requests.get(url, headers=headers)
+        return response
+
     try:
         response = requests.get(url, headers=headers)
     except (requests.exceptions.ConnectionError,
             requests.exceptions.ConnectTimeout):
         print(
-            "[!] There was an error connecting to the ProtonMail API.\n"
+            "[!] There was an error connecting to the ProtonVPN API.\n"
             "[!] Please make sure your connection is working properly!"
         )
-        logger.debug("Error connecting to ProtonMail API")
+        logger.debug("Error connecting to ProtonVPN API")
```

Comment:
Why is this `logger.debug` and not `logger.error`?

Reply:
I've created #48 for this to keep track of it. This should be done program wide in another commit